thirdparty/openapi3/schemas/UpdateCredentialRequest.yaml

```diff
@@ -0,0 +1,35 @@
+title: UpdateCredentialRequest
+type: object
+description: |
+  A credential used to allow a user to prove their identity and access
+  to an account with a DFSP.
+
+  UpdateCredentialRequest is a special formatting of the credential to allow us to be
+  more explict about the `status` field - it should only ever be PENDING when updating
+  a credential.
+properties:
+  id:
+    type: string
```

Comment:
question: is this a `CorrelationId`?

Reply:
Nah I don't think so - see other comment about the `credential.id`